YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -750,20 +769,34 @@ bool Repository::rebase(const std::string& upstream, const std::string& destinat
 
 bool Repository::add_file_to_index(const std::string& path)
 {
+    fs::path p = path;
+    if (!include_idb_ &&
+        (path == get_current_idb_name() ||
+        path == get_original_idb_name())) {
+        p.replace_extension(".idx_file");
+        std::fstream f;
+        f.open(p, std::fstream::out);
+        f << "DO NOT DELETE!" << std::endl;
+    }
     try
     {
-        repo_.add_file(path);
+        repo_.add_file(p);
```

Comment:
does not compile on msvc because add_file take a string and fs::path is not implicitely convertible to std::string

Reply:
fixed in f4e387aa68d432c5dc6030e0f89dc6e71f0760e2